tests/python/pants_test/backend/jvm/tasks/test_properties.py

```diff
@@ -19,12 +19,7 @@ def test_empty(self):
         self.assertLoaded("", {})
         self.assertLoaded(" ", {})
         self.assertLoaded("\t", {})
-        self.assertLoaded(
-            """
-
-    """,
-            {},
-        )
+        self.assertLoaded("\n", {})
```

Comment:
This test definitely doesn't have the `dedent` call, is preserving whitespace here necessary?

Reply:
I don't think so, given that the above test cases already check that we properly handle `" "` and `"\t"`. I think the intent of this particular assertion was simply that we properly handle new lines.